ci/prepare.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+
+export ERLANG_VERSION="17.1"
+export ELIXIR_VERSION="v1.0.4"
+
+# If you have a elixir_buildpack.config, do this instead:
+#export ERLANG_VERSION=$(cat elixir_buildpack.config | grep erlang_version | tr "=" " " | awk '{ print $2 }')
+#export ELIXIR_VERSION=v$(cat elixir_buildpack.config | grep elixir_version | tr "=" " " | awk '{ print $2 }')
```

Comment:
extracting the version numbers from the `elixir_buildpack.config` from now on. https://github.com/philippkueng/librarian/commit/dbb28c15158e9347851844da67e49b17270f3e49 thank you for pointing it out